pkg/controllers/cloud/virtualmachine_controller.go

```diff
@@ -53,6 +55,7 @@ func (r *VirtualMachineReconciler) Reconcile(_ context.Context, req ctrl.Request
 		accessor.SetName(req.Name)
 		accessor.SetNamespace(req.Namespace)
 	}
+	r.Log.V(0).Info("Debug: Dumping ", "vm", virtualMachine)
```

Comment:
nit: remove this

Reply:
Tried removed most of the unnecessary logs. If there are any pending, will address them in the final version.